src/test/java/core/basesyntax/db/DaoServiceHashMapTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.db;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DaoServiceHashMapTest {
+    private static final String DEFAULT_FRUIT = "apple";
+    private static final Integer VALUE_POSITIVE = 100;
+    private static final Integer VALUE_NEGATIVE = -100;
+    private static final Integer VALUE_ZERO = 0;
+    private static final Integer VALUE_NULL = null;
+    private static final String ADDITIONAL_FRUIT = "banana";
+    private static final DaoService<String, Integer> STORAGE
```

Comment:
```suggestion
    private static final DaoService<String, Integer> storage
```
this is not a magic string or number